queue_services/entity-bn/src/entity_bn/worker.py

```diff
@@ -49,10 +50,14 @@
 from entity_bn.exceptions import BNException, BNRetryExceededException
 
 
+flags = Flags()  # pylint: disable=invalid-name
 APP_CONFIG = config.get_named_config(os.getenv('DEPLOYMENT_ENV', 'production'))
 FLASK_APP = Flask(__name__)  # pragma: no cover
 FLASK_APP.config.from_object(APP_CONFIG)
-db.init_app(FLASK_APP)
+init_db(FLASK_APP)
+
+if FLASK_APP.config.get('LD_SDK_KEY', None):
```

Comment:
Not sure if we wanna have this check or not. 
Legal-api doesn't have this check https://github.com/bcgov/lear/blob/feature-db-versioning/legal-api/src/legal_api/__init__.py#L58C3-L60C24
But Emailer has this:
https://github.com/bcgov/lear/blob/feature-db-versioning/queue_services/entity-emailer/src/entity_emailer/worker.py#L77C1-L78C30

Reply:
I think, it's a good idea, so that we can control using FF or not by changing .env without touching the code